dedibox-network/rpn/how-to/use-rpn-san-esxi.mdx

```diff
@@ -0,0 +1,180 @@
+
+### Enable SSH on your ESXi
+
+1. Start the vSphere client or log in using the web interface. 
+
+2. Click the Configuration tab (1), then click **Security Profile** (2) in the software panel.
+
+  <Lightbox src="scaleway_esxi-san-en-01.png" /> 
+
+3. Click **Properties** in the Services section.
+
+  <Lightbox src="scaleway_esxi-san-en-02.png" /> 
+
+4. Select SSH (1) then click options (2). Select **Start and stop manually** (3) and launch the service manually (4).
+
+  <Lightbox src="scaleway_esxi-san-en-03.png" /> 
+
+### Configure ESXi to access the RPN
+
+#### Configuration in the ESXi interface
+
+1. Go to **Configuration** > **Networking** (1), then, click **Add Networking** (2).
+
+  <Lightbox src="scaleway_esxi-san-en-04.png" /> 
+
+2. Create a new VMKernel named **RPN**, attached to the RPN network card.
+
+  <Lightbox src="scaleway_esxi-san-en-05.png" /> 
+
+  <Lightbox src="scaleway_esxi-san-en-06.png" /> 
+
+For the next step, validate the windows by choosing **Automatic recovery** for the IP addressing.
+
+<Message type="important">
+
+The network will go down, because the virtual MAC is not the same as the physical one (for security reasons).
+
+</Message>
+
+#### Configuration of the MTU 9000
+
+1. Go to **Configuration** > **Networking** > **Properties...** (of vSwitch1).
+
+  <Lightbox src="scaleway_esxi-san-en-07.png" /> 
+
+2. Select the vSwitch (1) and edit it (2).
+
+  <Lightbox src="scaleway_esxi-san-en-08.png" /> 
+
+3. Set the MTU to to **9000**, then confirm and quit.
+
+  <Lightbox src="scaleway_esxi-san-en-09.png" /> 
+
+### ESXi configuration
+
+1. Edit the configuration file of ESXi
+  ```
+  nano /etc/vmware/esx.conf
+  ```
+
+2. Locate the number of the child in the RPN portgroup (in this example: 0001)
+  ```
+  /net/vmkernelnic/child[0001]/portgroup = "RPN"
+  ```
+3. Locate the MAC of the network card connected to the RPN (in this example: `XX:YY:ZZ:AA:BB:CC`)
+  ```
+  /net/pnic/child[0001]/mac = "XX:YY:ZZ:AA:BB:CC”
+  ```
+
+4. Replace the MAC of the VMKernel RPN (see the number of the child) with the MAC of the network card connected to the RPN
+  ```
+  /net/vmkernelnic/child[0001]/mac = "XX:YY:ZZ:AA:BB:CC”
+  ```
+
+5. Reboot the ESXi to take the changes into effect.
+
+6. Add a route for the RPN subnet via the IP of its gateway. [Find your RPN gateway.](/dedibox-network/rpn/how-to/find-rpn-gateway)
+  ```
+  esxcfg-route -a 10.88.0.0/13 [PUT HERE YOUR RPN GATEWAY]
+  ```
+
+  <Message type="important">
+
+  You need to add the route to `/etc/rc.`local` to keep it after a reboot.
+  For the version ESXi 5.1 the file is located at `/etc/rc.local.d/local.sh`.
+
+  </Message>
+
+7. Ping the RPN SAN to check your configuration.
+  ```
+  # ping san-1.rpn.online.net
+  PING 10.90.254.2 (10.90.254.2): 56 data bytes
+  64 bytes from 10.90.254.2: icmp_seq=0 ttl=61 time=0.241 ms
+  64 bytes from 10.90.254.2: icmp_seq=1 ttl=61 time=0.241 ms
+  ```
+  <Message type="important">
```

Comment:
check if everything is ok with the code from this point forward, as everything is appearing as blue